xml/System.Data.Common/DbBatchCommandCollection.xml

```diff
@@ -369,10 +371,8 @@
         <Parameter Name="item" Type="System.Data.Common.DbBatchCommand" />
       </Parameters>
       <Docs>
-        <param name="item">The object to remove from the <see cref="T:System.Collections.Generic.ICollection`1" />.</param>
-        <summary>Removes the first occurrence of a specific object from the <see cref="T:System.Collections.Generic.ICollection`1" />.</summary>
-        <returns>
-          <see langword="true" /> if <paramref name="item" /> was successfully removed from the <see cref="T:System.Collections.Generic.ICollection`1" />; otherwise, <see langword="false" />. This method also returns <see langword="false" /> if <paramref name="item" /> is not found in the original <see cref="T:System.Collections.Generic.ICollection`1" />.</returns>
+        <param name="value">The <see cref="T:System.Data.Common.DbBatchCommand" /> object to remove.</param>
+        <summary>Removes the specified <see cref="T:System.Data.Common.DbBatchCommand" /> object from the collection.</summary>
```

Comment:
Why was the `<returns>` section removed here?

Reply:
Good catch, this was a mistake. Brought it back and aligned with List.Remove.